packages/node_modules/@webex/widget-space/src/container.js

```diff
@@ -11,6 +11,8 @@ import LoadingScreen from '@webex/react-component-loading-screen';
 import Timer from '@webex/react-component-timer';
 import ErrorDisplay from '@webex/react-component-error-display';
 
+import {CONVERSATION_URL} from '@webex/plugin-meetings/dist/constants';
```

Comment:
i dont think you need dist if we are properly exploring the const in the plugin meetings

Reply:
Will check that @arun3528

Reply:
Done. Thanks @arun3528